Fix proxmox with offline node

aseemann opened the pull request:
Hi, 

this fixes that no information shown for proxmox if one node of the cluster is turned of for some reason. 

Feel free to check and merge.

mvdkleijn commented:
Please correct the issues mentioned in the codesniffer check. Other than that LGTM.

aseemann commented:
Hi @mvdkleijn, 

I've fixed the errors. I also added the ident config for PHP files to the .editorconfig, so IDE's supporting it will automatically use the correct style.

mvdkleijn commented:
Thanks! (sorry for the delayed response, holidays here)
LGTM. Merging.
